### ISSUE TITLE
ProudCity footer link color

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#1861]
Looks like there's an issue with the link color in the ProudCity band.

The link should be white (or something more accessible)?

See Lucas: https://www.lucastexas.us/

**Comment by kevindherman** [comment]
@lukefretwell @aschmoe is this because of the colors that Lucas specifically picked or something larger?

**Comment by lukefretwell** [comment]
@kevindherman something larger

**Comment by curtismchale** [comment]
@lukefretwell looking at this now many of the sites have white links as expected. Do you have more examples of themes with improper link colours?

It looks like something with the ProudCity Patterns library. I know that ends up in many spots so I have a question out to Alex about the best ways to update that.

**Comment by aschmoe** [comment]
@curtismchale there are a couple of things going on.

1. We analyze the colors being used for backgrounds then apply classes to the body accordingly:  https://github.com/proudcity/wp-proud-theme/blob/master/lib/extras.php#L72
2. We have code in the individual zone files that apply that color: https://github.com/proudcity/proudcity-patterns/blob/master/app/pattern-scss/_local-variables.scss#L83
3. The color is defined in: https://github.com/proudcity/proudcity-patterns/blob/master/app/pattern-scss/_local-variables.scss#L83

@lukefretwell I am seeing those particular colors as within accessibility standards however: 
<img width="1148" alt="Screen Shot 2022-05-10 at 1 53 41 PM" src="https://user-images.githubusercontent.com/1634686/167720076-38360ee8-7d78-4692-b774-6e3659e0e0a5.png">

As for the patterns, as long as you're pulling and pushing with gulp in the theme, we should be all good -- 
 https://github.com/proudcity/wp-proud-theme#working-with-proudcity-patterns (but maybe you were having trouble with those with your version of node?)

**Comment by aschmoe** [comment]
@lukefretwell and by extension, white in this instance would totally fail

<img width="833" alt="Screen Shot 2022-05-10 at 1 58 18 PM" src="https://user-images.githubusercontent.com/1634686/167720687-bad25b10-3dff-4285-993b-7710a722d898.png">

**Comment by aschmoe** [comment]
@lukefretwell @curtismchale I was confused:

oh, we're talking not the footer but the powered by... looks like we just need to extend the color here to links as well -- https://github.com/proudcity/proudcity-patterns/blob/master/app/pattern-scss/_powered-by.scss#L16

**Comment by lukefretwell** [comment]
@curtismchale @aschmoe sorry, yes, was talking about the 'powered by ProudCity' link.

**Comment by curtismchale** [comment]
![CleanShot 2022-05-12 at 08 15 45@2x](https://user-images.githubusercontent.com/74561/168109332-aa8ad346-dd8a-4221-8017-cc533016d81e.png)

@lukefretwell nothing wrong with our theme there was a link tag without a matching closing tag so the expected CSS didn't get applied to the powered by link. 

I fixed the HTML and this looks like it's working fine now. I'm closing the tag since it doesn't really need to be mentioned in a release.
